modules/sugar/src/test/ts/browser/VisibilityTest.ts

```diff
@@ -19,12 +17,13 @@ UnitTest.test('VisibilityTest', () => {
   Css.set(c, 'display', 'none');
   Assert.eq('', false, Visibility.isVisible(c));
 
-  const s = SugarElement.fromTag('span');
-  Assert.eq('', false, Visibility.isVisible(s));
-
-  Insert.append(SugarBody.body(), s);
-  const expected = PlatformDetection.detect().browser.isFirefox();
-  Assert.eq('', expected, Visibility.isVisible(s)); // tricked you! height and width are zero == hidden
+  // TODO: Disabled since it is flaking on Chrome sometimes it's hidden sometimes its not
```

Comment:
Do we have a Notion or JIra we can link this to?

Reply:
Added a ticket reference so that we can investigate this further.